core/mondoo-vmware-vulnerability.mql.yaml

```diff
@@ -0,0 +1,48 @@
+policies:
+  - uid: mondoo-vmware-vulnerability
+    name: VMware vCenter Vulnerability Policy by Mondoo
+    version: "1.0.0"
+    authors:
+      - name: Mondoo, Inc
+        email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: vmware,vmware-vsphere
+      mondoo.com/category: security
+    docs:
+      desc: |
+        ## Overview
+
+        Mondoo OpenSSL VMware vCenter Policy checks for vulnerable vCenter/ESXi configuration. It should be used in combination with Mondoo's Platform Vulnerability Management to identify missing patches.
```

Comment:
What is `Mondoo's Platform Vulnerability Management`?

Reply:
<img width="1223" alt="Screenshot 2023-02-08 at 00 52 37" src="https://user-images.githubusercontent.com/1178413/217393511-4fdca829-ae4b-4f95-b755-668a618e7b47.png">

Reply:
Shouldn't that be calling out the policy name then?

Reply:
Agreed. Thank you @tas50